src/Microsoft.TemplateEngine.Edge/Template/TemplateCreator.cs

```diff
@@ -106,7 +106,6 @@ public IReadOnlyCollection<IFilteredTemplateInfo> List(bool exactMatchesOnly, pa
                 IComponentManager componentManager = _environmentSettings.SettingsLoader.Components;
                 creationResult = await template.Generator.CreateAsync(_environmentSettings, template, templateParams, componentManager, targetDir).ConfigureAwait(false);
                 sw.Stop();
-                _environmentSettings.Host.OnTimingCompleted("Content generation time", sw.Elapsed);
```

Comment:
I thought about this, but I'd like to keep it here in case we add a timing switch where we can read back timing data (especially from the Timing.Over calls)